src/protocols/BaseProtocolAdapter.ts

```diff
@@ -25,7 +27,10 @@ export abstract class BaseProtocolAdapter {
 
   abstract getAmountPrecision(args?: any): number;
 
-  abstract getCoinSymbol(getShort?: boolean): string;
+  /**
+   * selector='protocol' by default
+   */
+  abstract getCoinSymbol(selector?: ProtocolSymbolTypes): string;
```

Comment:
Maybe we could have 3 different methods for each symbol type? Or even store it as a class property instead of getter as this is a thing that doesn't change.